packages/experimental/AppearanceAdditions/ios/FRNAppearanceAdditions.m

```diff
@@ -105,23 +107,24 @@ - (dispatch_queue_t)methodQueue
 - (void)startObserving {
     _hasListeners = YES;
     
-    // Note that [UITraitCollection currentTraitCollection] always returns the same default trait collection,
-    // presumably because FRNAppearanceAdditions isn't a view, so it never gets updated with the right traitCollection
-    // (which happens when a view gets added to the view hierachy). In order to get the right trait collection,
-    // we need to access a view that's been added to the view hierarchy
-    UIViewController *viewControllerWithInitialTraitCollection = RCTPresentedViewController();
-    UITraitCollection *initialTraitCollection;
+    _rootTagHorizontalSizeClassMap = [NSMutableDictionary new];
+    _rootTagUserInterfaceLevelMap = [NSMutableDictionary new];
+    _rootTagAccessibilityContrastMap = [NSMutableDictionary new];
     
-    if (viewControllerWithInitialTraitCollection != nil) {
-        initialTraitCollection = [viewControllerWithInitialTraitCollection traitCollection];
-    } else {
-        initialTraitCollection = [UITraitCollection currentTraitCollection];
+    for (UIWindow *window in RCTSharedApplication().windows)
+    {
```

Comment:
This class has been keeping curly braces on the same line as the `for` or `if` statement. Do we have a single coding convention for FURN? If not, let's at least make sure the whole file is internally consistent.

Reply:
We haven't added a clang format file yet.

Reply:
Fixed, sorry the { on new line was the convention on a previous team